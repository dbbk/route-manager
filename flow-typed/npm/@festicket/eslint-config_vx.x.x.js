// flow-typed signature: 564086c0274f153b4c37ac5c954f219e
// flow-typed version: <<STUB>>/@festicket/eslint-config_v^1.1.5/flow_v0.63.1

/**
 * This is an autogenerated libdef stub for:
 *
 *   '@festicket/eslint-config'
 *
 * Fill this stub out by replacing all the `any` types.
 *
 * Once filled out, we encourage you to share your work with the
 * community by sending a pull request to:
 * https://github.com/flowtype/flow-typed
 */

declare module '@festicket/eslint-config' {
  declare module.exports: any;
}

/**
 * We include stubs for each file inside this npm package in case you need to
 * require those files directly. Feel free to delete any files that aren't
 * needed.
 */


// Filename aliases
declare module '@festicket/eslint-config/index' {
  declare module.exports: $Exports<'@festicket/eslint-config'>;
}
declare module '@festicket/eslint-config/index.js' {
  declare module.exports: $Exports<'@festicket/eslint-config'>;
}
