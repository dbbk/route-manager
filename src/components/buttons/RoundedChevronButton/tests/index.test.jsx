// @flow

import React from 'react';
import { shallow } from 'enzyme';
import { RoundedChevronButton } from '@festicket/react-ui-components';

describe('<RoundedChevronButton />', () => {
  test('with initial props', () => {
    const tree = shallow(<RoundedChevronButton />)
      .first()
      .shallow();
    expect(tree).toMatchSnapshot();
  });
});
