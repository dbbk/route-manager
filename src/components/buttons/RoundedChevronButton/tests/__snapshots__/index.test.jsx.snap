// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RoundedChevronButton /> with initial props 1`] = `
.c2 {
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #8C64CB;
}

.c1 {
  display: inline-block;
  -webkit-text-align: center;
  text-align: center;
  cursor: pointer;
  font-weight: bold;
  border-radius: 0;
  min-height: 50px;
  min-width: 200px;
  min-height: 50px;
  padding: 15px 30px;
  color: #A2A4AD;
  background-color: rgba(36,37,41,0.05);
}

.c1:hover {
  background-color: rgba(36,37,41,0.1);
  color: #9497a1;
}

.c3 {
  height: 18px;
  fill: #A2A4AD;
}

.c0 {
  font-size: 18px;
  border-radius: 100%;
  width: auto;
  min-width: 0;
  min-height: 0;
  padding: 0;
  width: 28px;
  height: 28px;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

<a
  className="c0 c1 c2"
  element="a"
  size="regular"
  target="_blank"
  to="#"
  variant="standard"
  weight="bold"
>
  <svg
    className="c3"
    color="grey"
    viewBox="0 0 18 18"
  >
    <path
      d="M4.8 3.3L10.4 9l-5.6 5.7 1.4 1.4 7-7.1-7-7.1z"
    />
  </svg>
</a>
`;
